src/modules/configuration/tests/ConfigurationTests.cpp

```diff
@@ -0,0 +1,381 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <gtest/gtest.h>
+#include <version.h>
+#include <Mmi.h>
+#include <CommonUtils.h>
+#include <Configuration.h>
+
+using namespace std;
+
+class ConfigurationTest : public ::testing::Test
```

Comment:
Should unit tests be placed in `namespace Tests { ... }` like other modules?

Reply:
The Test namespace is used in many places, including for several modules test:  

src\common\tests\CommonUtilsUT.cpp(20):class CommonUtilsTest : public ::testing::Test
src\modules\adhs\tests\AdhsTests.cpp(12):class AdhsTest : public ::testing::Test
src\modules\commandrunner\tests\CommandRunnerTests.cpp(14):    class CommandRunnerTests : public ::testing::Test
src\modules\configuration\tests\ConfigurationTests.cpp(12):class ConfigurationTest : public ::testing::Test
src\modules\deliveryoptimization\tests\DeliveryOptimizationTests.cpp(12):class DeliveryOptimizationTest : public ::testing::Test
src\modules\deviceinfo\tests\DeviceInfoTests.cpp(12):class DeviceInfoTest : public ::testing::Test
src\modules\firewall\tests\FirewallTests.cpp(126):    class FirewallTests : public ::testing::Test
src\modules\test\RecipeInvoker.h(12):class RecipeFixture : public ::testing::Test {};
src\modules\tpm\tests\TpmTests.cpp(57):    class TpmTests : public ::testing::Test
src\platform\tests\ManagementModuleTests.cpp(15):    class ManagementModuleTests : public ::testing::Test
src\platform\tests\ModulesManagerTests.cpp(26):    class ModuleManagerTests : public ::testing::Test
src\platform\tests\MpiServerTests.cpp(14):    class MpiServerTests : public ::testing::Test {};
src\platform\tests\MpiTests.cpp(18):    class MpiTests : public ::testing::Test

Reply:
Let's discuss this offline, we'll want to uniformize the namespace for all tests. My vote is for "Test" as this is one namespace, not many.